subprojects/core-api/src/main/java/org/gradle/api/file/ProjectLayout.java

```diff
@@ -47,13 +47,24 @@ public interface ProjectLayout extends FileSystemLayout {
 
     /**
      * {@inheritDoc}
+     *
+     *  <p>
+     *  File system locations based on relative paths will be
+     *  resolved against this layout's reference location, as defined by {@link #getProjectDirectory()}.
+     *  </p>
+     *
      */
     @Override
     Provider<RegularFile> file(Provider<File> file);
 
     /**
      * {@inheritDoc}
      *
+     * <p>
+     * File system locations based on relative paths will be
+     * resolved against this layout's reference location, as defined by {@link #getProjectDirectory()}.
+     * </p>
+     *
```

Comment:
❔  Would the javadoc produce the same result if we drop the empty line 62 and the `</p>` line 66?

Reply:
Yup, closing paragraph tags are optional, and new lines have no effect on javdoc output. Are you suggesting we did that?

Reply:
✅

Reply:
If it uses fewer lines of code, does the same thing, and does not hurt readability, I would take it any day! 😄